core/dbt/compilation.py

```diff
@@ -142,11 +154,77 @@ def write_graph(self, outfile: str, manifest: Manifest):
         include all nodes in their corresponding graph entries.
         """
         out_graph = self.graph.copy()
-        for node_id in self.graph.nodes():
+        for node_id in self.graph:
             data = manifest.expect(node_id).to_dict(omit_none=True)
             out_graph.add_node(node_id, **data)
         nx.write_gpickle(out_graph, outfile)
 
+    def resolve_graph(self, manifest: Manifest):
```

Comment:
We haven't been doing this consistently (I include myself in this), but the idiomatic way to do this type hint should probably include `-> None`.  Super minor, but mypy will carp about it when (if) we ever crank up the strictness.